### ISSUE TITLE
benchmark code crashing constantly on first write

### ISSUE THREAD

**Issue by cmeiklejohn** [SyncFree/antidote#114]
The benchmark code crashes immediately for the first write for every key -- this causes the benchmarker to restart threads and it's skewing the benchmark and the code can't handle writing the first object correctly.

For example:

```
11:58:53.690 [error] CRASH REPORT Process <0.18262.0> with 0 neighbours exited with reason: {error,{badmatch,{error,no_snapshot}},[{antidote_pb_counter,process,2,[{file,"src/antidote_pb_counter.erl"},{line,74}]},{riak_api_pb_server,process_message,4,[{file,"src/riak_api_pb_server.erl"},{line,388}]},{riak_api_pb_server,connected,2,[{file,"src/riak_api_pb_server.erl"},{line,226}]},{riak_api_pb_server,decode_buffer,2,[{file,"src/riak_api_pb_server.erl"},{line,364}]},{gen_fsm,handle_msg,7,[{file,"gen_fsm.erl"},{line,505}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]} in gen_fsm:terminate/7 line 622
```

To reproduce: 
- Run the benchmark on any branch (evaluation, master, ec_antidote..)
- It will immediately crash because it can't handle the first write to a given key, because no previous snapshots exist.

@marsleezm @bieniusa This needs to be fixed immediately.

**Comment by aletomsic** [comment]
This relates to the no_snapshot fix.




Alejandro Z. Tomsic

**Comment by cmeiklejohn** [comment]
@marsleezm Is there a branch that contains this fix?

**Comment by aletomsic** [comment]
Just for clarification, the issue that causes every first update to get {error, no_snapshot} should be fixed in fix #105.
The problem that I believe @marsleezm fixed, is handling such error at the pb client.

**Comment by marsleezm** [comment]
Sorry I don't have my computer.. It's for sure solved in evaluation_pb_tx
branch with latest commit of antidote_pb, but I don't remember which repo
has the necessary fix. maybe both.
